### ISSUE TITLE
`tensorflow-cl` branch build on ubuntu 16.04 not quite working, if someone has a moment to take a look?

### ISSUE THREAD

**Issue by hughperkins** [hughperkins/tensorflow-cl#31]
Hi,

The build on ubuntu 16.04 isnt quite working. I broke it by "upgrading" the instructions to use bazel 0.4.5, instead of 0.3.2. It fails with an issue with not finding `protoc`. If someone has a moment to take a look, would be much appreciated :-) . (I'm busy fixing Mac/Radeon stuff at the moment, personally)

**Comment by hughperkins** [comment]
(basically I got as far as trying things like building protoc first, ie:

```
bazel build @protobuf//:protoc
mkdir -p bazel-out/host/bin/external/protobuf
ln -sf $PWD/bazel-bin/external/protobuf/protoc bazel-out/host/bin/external/protobuf/protoc
```

... but seems not to quite work?  I think a workaround is to `apt-get install` protobuf-c-compiler etc, but I think it'd be cleaner to use the tensorflow one. (And even cleaner to figure out why this isnt being built automatically; I'm fairly sure it used ot be, using the old bazel).

**Comment by hughperkins** [comment]
addressed in 1f023db , I think
